.ui/library/lviNutritions.cpx

```diff
@@ -0,0 +1,41 @@
+{
+	"components": [
```

Comment:
generate appIcon and use not transparent image for titlelayotu

Reply:
Miss